libs/mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/media/rest_v11_sites_158396482_media_new.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
This file also isn't needed anymore since you change to using an image from the library.

Reply:
Same question as above